conf/README.md

```diff
@@ -1,9 +1,9 @@
 # Configuration of webwork2
 
 Do not directly modify any of the files in the initial git clone of webwork2.  If a filename contains `.dist` then the
-file is intended to be copied to a file by the same name without `.dist` and the copy modified as needed.  If a filename
-does not contain `.dist` then it should not be modified.  Copies of the `.dist` files will remain unchanged when
-webwork2 is upgraded, but the modifications to the other files will be lost or cause conflict.
+file is intended to be copied to a file by the same name without `.dist` and the copy modified as needed.  Specific
+instructions are given below.  If a filename **does** contain `.dist` then it should **not** be modified.  Copies of
+the `.dist` files will update when webwork2 is upgraded.
```

Comment:
I don't entirely agree with the wording "will update when webwork2 is upgraded." If there are local modifications to the file that isn't what actually will happen.  `git` will refuse to do the `pull` in that case, or you must force `git` to do so, or first revert any local changes.